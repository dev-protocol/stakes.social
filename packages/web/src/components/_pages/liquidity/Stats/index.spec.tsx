// import React from 'react'
// import { render } from '@testing-library/react'
import { Stats } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/_pages/liquidity/geyser/hooks.ts')
jest.mock('src/fixtures/_pages/liquidity/uniswap-pool/hooks.ts')

describe(`${Stats.name}`, () => {
  test('Snapshot', () => {
    // const component = render(<Stats geyserAddress="0xdummy" />)
    // const tree = component.baseElement
    // expect(tree).toMatchSnapshot()
  })
})
