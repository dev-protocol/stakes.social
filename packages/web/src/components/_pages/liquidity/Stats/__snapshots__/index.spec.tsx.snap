// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-bdvvtL ixtLKA"
    >
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Total rewards
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            1e-13
          </span>
          <span
            class="ant-statistic-content-suffix"
          >
            DEV
          </span>
        </div>
      </div>
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Total deposits
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            1e-13
          </span>
          <span
            class="ant-statistic-content-suffix"
          >
            USD
          </span>
        </div>
      </div>
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Locked rewards
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            <span
              class="ant-statistic-content-value-int"
            >
              
              0
            </span>
            
          </span>
          <span
            class="ant-statistic-content-suffix"
          >
            DEV
          </span>
        </div>
      </div>
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Unlocked rewards
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            <span
              class="ant-statistic-content-value-int"
            >
              
              0
            </span>
            
          </span>
          <span
            class="ant-statistic-content-suffix"
          >
            DEV
          </span>
        </div>
      </div>
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          End of the Program
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            1/2/1970
          </span>
        </div>
      </div>
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Reward unlock rate
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            2.592e-12
          </span>
          <span
            class="ant-statistic-content-suffix"
          >
            DEV
          </span>
        </div>
      </div>
    </div>
  </div>
</body>
`;
